3pl_logistic_company/data/portal_templates.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <template id="portal_my_home_products" name="Your products" inherit_id="portal.portal_my_home">
```

Comment:
add an image on the portal 
<img width="624" height="167" alt="image" src="https://github.com/user-attachments/assets/00313a61-5e7e-48c4-b7ca-9429471c7409" />